support-frontend/test/selenium/util/PostDeployTestUser.scala

```diff
@@ -24,3 +28,31 @@ class PostDeployTestUser(driverConfig: DriverConfig) extends TestUser {
   val username = testUsers.generate()
   addTestUserCookies(username)
 }
+
+class PostDeployTestUserSubs(driverConfig: DriverConfig) extends TestUser {
+
+  private val client = new OkHttpClient()
+  private val requestToIdapi = new Request.Builder()
+    .url(Config.idapiNewTestUserUrl)
+    .addHeader(Config.idapiClientAccessTokenName, Config.idapiClientAccessTokenSecret)
+    .post(RequestBody.create(MediaType.parse("application/json"), "{}"))
+    .build()
+  private val response = client.newCall(requestToIdapi).execute()
+  private val responseBodyAsJson = parse(response.body().string()).getOrElse(Json.Null)
+  private val cursor = responseBodyAsJson.hcursor
+
+  private def addTestUserCookies(testUsername: String) = {
+    driverConfig.addCookie(name = "pre-signin-test-user", value = testUsername)
+    driverConfig.addCookie(name = "_test_username", value = testUsername, domain = Some(Config.guardianDomain))
+    driverConfig.addCookie(name = "_post_deploy_user", value = "true") // This enables the tests to use the mocked payment services
+    driverConfig.addCookie(name = "GU_TK", value = "1.1") //To avoid consent banner, which messes with selenium
+    for (n <- 0 until 3) {
+      val cookie_name = cursor.downField("values").downN(n).downField("key").as[String].right.get
+      val cookie_value = cursor.downField("values").downN(n).downField("value").as[String].right.get
+      driverConfig.addCookie(name = cookie_name, value = cookie_value)
+    }
+  }
+
+  val username = "postDeployTestUser"
```

Comment:
why can this just be hard coded, I assumed it would have to be a proper test user string?

Reply:
Reverted back to using a test user string, which is needed for the Contributions checkouts tests.